services/rollout-service/pkg/cmd/server.go

```diff
@@ -103,6 +110,21 @@ func (config *Config) RevolutionConfig() (revolution.Config, error) {
 	}, nil
 }
 
+func (config *Config) DataDogDoraConfig() (datadogdora.Config, error) {
```

Comment:
helm chart adaptations are missing. Please add them here https://github.com/freiheit-com/kuberpult/blob/main/charts/kuberpult/templates/rollout-service.yaml
Basically, every env var needs to be in the helm chart (and make sure to keep everything backwards compatible)